setup.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+import sys
+from setuptools import setup
+from warnings import warn
+
+if sys.version_info < (3, 7, 0):
+    # does 2.7 understand f strings?
+    # warn(f"Python version 3.7 or later is required for crdch_model.  Current version: {sys.version_info}")
+    warn("Python version 3.7 or later is required for crdch_model.  Current version: " + str(sys.version_info))
+    sys.exit(1)
+
+setup(
+    setup_requires=['pbr'],
+    pbr=True,
+)
```

Comment:
The `setup.py` file looks good, it looks like most of the attributes were captured in the `setup.cfg` file.🚀

I have a quick question here, can we parse the model version information out from [crdch_model.yaml](https://raw.githubusercontent.com/cancerDHC/ccdhmodel/issue-69-pypi/model/schema/crdch_model.yaml), and add a `version` parameter to the `setup()` method?

Reply:
I hadn't thought of that. Would that lead to the model's version automatically being displayed at PyPI?

@gaurav, what do you think?

Reply:
Yup, that would automatically pull the version number from the YAML file, populate the `setup.py` file, and push the same version number to PyPI too.

Reply:
You should also add a `name` parameter to the `setup.py` file. This way PyPI will know what the name of the package being deployed to it will be.

Reply:
Sorry, I'm probably going to be asking a lot of stupid questions. PyPI alredy does know what the package should be called. Do you know where it might be getting that?

Reply:
Yup, so currently it's picking up the package name from [setup.cfg](https://github.com/cancerDHC/ccdhmodel/blob/issue-69-pypi/setup.cfg#L3) `name` attribute. I didn't see that myself, apologies. I think we can skip the `name` parameter in `setup.py` in this case then.